.storybook/preview.js

```diff
@@ -1,5 +1,9 @@
+/*
+  * The addParameters and addDecorator APIs to add global decorators and parameters, exported by the various frameworks (e.g. @storybook/react) and @storybook/client were deprecated in 6.0 and have been removed in 7.0.
+
+Instead, use export const parameters = {}; and export const decorators = []; in your .storybook/preview.js. Addon authors similarly should use such an export in a preview entry file (see Preview entries).
+  * */
```

Comment:
I appreciate the explanation and it can happen in another PR but I think it would be fine to remove this as you noted in the description